codec-http2/src/test/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionEncoderTest.java

```diff
@@ -200,20 +201,6 @@ public Object answer(InvocationOnMock invocationOnMock) throws Throwable {
     }
 
     @Test
-    public void dataWriteAfterGoAwayShouldFail() throws Exception {
```

Comment:
@Scottmitch So I added those four tests https://github.com/netty/netty/pull/3516/files#diff-88de613b9690effaac3b56567ee67aa5R514 that "check" that we can send after a GO_AWAY was sent / received.